### PR TITLE
Disable no viewer mode

### PR DESCRIPTION
**Pull request by olegrise**
## Description
Disable no viewer mode

## Motivation and Context
https://trello.com/c/5qgtX1Mj/361-2-disable-no-viewer-on-chrome-os

## How Has This Been Tested?
Manually and with unit tests

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by olegrise** [comment]
@andrecardoso I created a draft PR for now. I don't like how "content-update" is handled here 38f1cf9436a1c7057faf77ebd7e9af16adec04a0. I wonder if you can take a look and see if you have better ideas. There is one use case in particular that is not working correctly. It's when schedule is updated while Player is offline. In this case Player misses the "content-update" message and continues showing old schedule even when it comes back online.

**Comment by olegrise** [comment]
@andrecardoso re. using `navigator.onLine` - here is what [MDN article](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/onLine) says about it

```
In Chrome and Safari, if the browser is not able to connect to a local area network (LAN) or a router, it is offline; all other conditions return true. So while you can assume that the browser is offline when it returns a false value, you cannot assume that a true value necessarily means that the browser can access the internet. You could be getting false positives, such as in cases where the computer is running a virtualization software that has virtual ethernet adapters that are always "connected." Therefore, if you really want to determine the online status of the browser, you should develop additional means for checking.
```

We use `navigator.onLine` in the [network-check](https://github.com/Rise-Vision/chrome-os-player/blob/3eeae71d5c43cb399f558ecc24a38cb5b76120a3/src/network-checks/index.js#L77) class during Player startup as a [first step](https://github.com/Rise-Vision/chrome-os-player/blob/3eeae71d5c43cb399f558ecc24a38cb5b76120a3/src/network-checks/index.js#L46) in the network check sequence which makes total sense, but using `navigator.onLine` in the [loadData](https://github.com/Rise-Vision/chrome-os-player/pull/247/files#diff-dccb525022b74455b7f414edbccc1bc1R28) for deciding if content should be loaded from cache is probably not a good idea.

**Comment by olegrise** [comment]
@andrecardoso please review

File: test/unit/scheduling/schedule-parser.js [hunk 1]
```diff
@@ -307,4 +307,67 @@ describe("Schedule Parser", () => {
     assert.equal(scheduleParser.hasOnlyNoViewerURLItems(data), false);
 
   });
+
+  it("should return true when shedule has temapltes only", () => {
```
Comment (andrecardoso) [inline test/unit/scheduling/schedule-parser.js]: Typo here: `templates`

File: test/unit/scheduling/schedule-parser.js [hunk 2]
```diff
@@ -307,4 +307,67 @@ describe("Schedule Parser", () => {
...
+          items: [templateItem1, templateItem2]
+        }
+      }
+    };
+
+    assert.equal(scheduleParser.hasOnlyHtmlTemplates(data), true);
+
+  });
+
+  it("should return false when shedule includes non temaplte items", () => {
```
Comment (andrecardoso) [inline test/unit/scheduling/schedule-parser.js]: Same

**Pull request merged**